src/Model/EventModel.php

```diff
@@ -40,10 +40,7 @@ public function __construct(array $config ) {
         );
         $this->properties = $config['properties'];
         // Can pass in unix timestamp if prefixed with '@'. Else just let it parse the date
-        $this->time = !empty($config['time']) ?
-            new DateTime(
```

Comment:
Couple of things here, since the class exists in Klaviyo\Model\EventModel namespace referencing new DateTime will throw an error since DateTime is not defined in that scope. It will need `use DateTime` specified at the top or instantiate the class as `new \DateTime` 

You are correct, passing in the timestamp with @ sign will return an object. Do we have any preference on the timezone or the format of the time?